floyd/cli/run.py

```diff
@@ -77,12 +77,12 @@ def process_data_ids(data_ids):
                       "\tfloyd run --data <namespace>/datasets/<dataset_name>:<mount_dir>\n"
                       "\tfloyd run --data <namespace>/datasets/<dataset_name>/<version>:<mount_dir>\n"
                       "\tfloyd run --data <namespace>/projects/<project_name>/<version>:<mount_dir>\n"
```

Comment:
Should this include the `/output` suffix?

Reply:
Yeah, great call!